src/background/mobileEvents.js

```diff
@@ -0,0 +1,30 @@
+import {Component} from "./component.js";
+
+export class MobileEvents extends Component {
+    async init() {
+        console.log("Creating port to mozacSecureProxy")
```

Comment:
I would do:

if (!constants.isAndroid) {
  return;
}

And remember to add: import {constants} from "./constants.js"